web/tools/webpack.config.ts

```diff
@@ -351,7 +382,7 @@ const createConfig = (
   // Optimize the bundle in production mode
   // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
   if (!devServer && optimize) {
-    config.plugins?.push(new optimize.AggressiveMergingPlugin())
+    config.plugins?.push(new optimize.AggressiveMergingPlugin(), new optimize.AggressiveSplittingPlugin({}))
```

Comment:
i get a deprecation warning
```
(node:14657) [DEP_WEBPACK_AGGRESSIVE_SPLITTING_PLUGIN] DeprecationWarning: AggressiveSplittingPlugin is deprecated in favor of SplitChunksPlugin
```

Reply:
Argh. This is not even required, because I added split chunks above.